src/labels/CustomLabels.labels

```diff
@@ -7264,6 +7280,166 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <protected>true</protected>
+        <shortDescription>Title for Create Constituent Records section in Get Started</shortDescription>
+        <value>Create Constituent Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button text for Connect Contacts to Organization checklist item</shortDescription>
+        <value>Create Affiliations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Connect Contact to Organization checklist item in Get Started</shortDescription>
+        <value>Affiliations in NPSP let you capture important connections between your constituents and related organizations such as work, school, or business. Connect a Contact to an Organization using Organization Affiliations.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for Connect Contacts to Organization checklist item in Get Started</shortDescription>
+        <value>Learn more about Affiliations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for Connect Contact to Organization checklist item in Get Started</shortDescription>
+        <value>Connect a Contact to an Organization with Affiliations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsBlueButton</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button text for Connect Two Contacts with Relationships checklist item</shortDescription>
+        <value>Create Relationships</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemConnectContactsRelationshipsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for Connect Two Contacts with Relationships checklist item</shortDescription>
+        <value>Relationships in NPSP let you to capture important connections between your constituents. Connect Contacts using the Relationships related list on the Contact records.</value>
```

Comment:
Small typo in this one. Please remove the word "to" before capture. It should be: Relationships in NPSP let you capture important connections between your constituents.